src/components/table/src/_table.scss

```diff
@@ -0,0 +1,178 @@
+.table {
```

Comment:
Potentially the child `.table__<thing>` could be `&__<thing>` which will reduce the amount of classnames outputted (e.g. `.table .table__caption {` would just be `.table__caption { `), but also specificity which could affect overriding things. 🤷🏻‍♂️

Reply:
Yep you're right.